pandas/errors/__init__.py

```diff
@@ -0,0 +1,60 @@
+# flake8: noqa
+
+""" expose public exceptions & warnings """
+
+from pandas._libs.tslib import OutOfBoundsDatetime
+
+
+class PandasError(Exception):
+    pass
```

Comment:
can we have a brief docstring to document what each of these error types are for?